src/zino/zino.py

```diff
@@ -51,7 +51,7 @@ def init_event_loop(args: argparse.Namespace):
         func=state.state.events.delete_closed_events,
         trigger="interval",
         minutes=30,
-        next_run_time=datetime.now(),
+        next_run_time=datetime.now() + timedelta(minutes=30),
```

Comment:
I believe you can drop this line completely. The first iteration of this job will always be scheduled to 30 minutes in the future unless you explicitly set a `next_run_time`.  This is why we set `next_run_time` in the other jobs, because we *do* want their first iteration to run immediately.